make sure the library name is the same no matter whether is built as
secure or not
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -343,15 +343,6 @@ else()
 endif()
 
 set(mi_basename "mimalloc")
-if(MI_SECURE)
-  set(mi_basename "${mi_basename}-secure")
-endif()
-if(MI_TRACK_VALGRIND)
-  set(mi_basename "${mi_basename}-valgrind")
-endif()
-if(MI_TRACK_ASAN)
-  set(mi_basename "${mi_basename}-asan")
-endif()
 string(TOLOWER "${CMAKE_BUILD_TYPE}" CMAKE_BUILD_TYPE_LC)
 if(NOT(CMAKE_BUILD_TYPE_LC MATCHES "^(release|relwithdebinfo|minsizerel|none)$"))
   set(mi_basename "${mi_basename}-${CMAKE_BUILD_TYPE_LC}") #append build type (e.g. -debug) if not a release version
